frontends/web/src/assets/exchanges/exchanges.json

```diff
@@ -0,0 +1,139 @@
+[{
```

Comment:
is this the final format? the first line looks out of place.
is this tabs? not sure how JSON files are formatted these days and don't really care :) but just make sure it's consistent with the rest of the app.